source/mongodump.txt

```diff
@@ -321,6 +321,22 @@ If you're using :ref:`views <views-landing-page>`:
 - To export the documents in a view, use
   :option:`--viewsAsCollections <mongodump --viewsAsCollections>`.
 
+Metadata Output
+~~~~~~~~~~~~~~~
+
+If :binary:`~bin.mongodump` outputs to a directory, the output contains
+collection metadata which includes the following: 
+
+- Collection name
+- Collection type
+- Creation options
```

Comment:
[question] Upon a quick google search, I couldn't find what "creation options" are. Is this referring to index creation options or something else?

Reply:
Reached out to Tim to clarify!

Reply:
The creation options are the fields specified in the "options" parameter when creating a collection (referenced [here](https://www.mongodb.com/docs/manual/reference/method/db.createCollection/#db.createcollection--)) . I changed the phrasing to "Collection configuration options"

Reply:
Perfect, thanks!